.github/workflows/Build.yml

```diff
@@ -30,29 +30,34 @@ jobs:
         CALVER="$( date -u '+%Y.%m.%d' )"
         SHA7="${GITHUB_SHA::7}"
         DOCKER_TAG=$SHA7
-        IMAGE_SPEC="${DOCKER_ORG}/${{ env.IMAGE }}:${DOCKER_TAG}"
         echo "DOCKER_TAG=${DOCKER_TAG}" >> $GITHUB_ENV
-        echo "IMAGE_SPEC=${IMAGE_SPEC}" >> $GITHUB_ENV
 
-    - name: Build and Publish to DockerHub
-      uses: elgohr/Publish-Docker-Github-Action@v4
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Login to Docker Hub
+      uses: docker/login-action@v2
       with:
-        name: ${{env.DOCKER_ORG}}/${{ env.IMAGE }}
         username: ${{ secrets.DOCKERHUB_USER }}
         password: ${{ secrets.DOCKERHUB_TOKEN }}
-        workdir: ${{ env.IMAGE }}
-        tags: "master, ${{env.DOCKER_TAG}}"
-    
-    - name: Also Push To quay.io
-      id: push-to-quay
-      uses: redhat-actions/push-to-registry@v2
+
+    - name: Login to Quay.io
+      uses: docker/login-action@v2
       with:
-        image: ${{env.DOCKER_ORG}}/${{ env.IMAGE }}
-        tags:  master ${{env.DOCKER_TAG}}
         registry: quay.io
-        username: ${{ secrets.QUAY_USERNAME }}
-        password: ${{ secrets.QUAY_PASSWORD }}
+        username: ${{ secrets.QUAY_USER }}
+        password: ${{ secrets.QUAY_TOKEN }}
 
+    - name: Build and push
+      uses: docker/build-push-action@v3
+      with:
+        context: ${{ env.IMAGE }}
+        push: true
+        tags: |
+          ${{env.DOCKER_ORG}}/${{ env.IMAGE }}:master
+          ${{env.DOCKER_ORG}}/${{ env.IMAGE }}:${{env.DOCKER_TAG}}
+          quay.io/${{env.DOCKER_ORG}}/${{ env.IMAGE }}:master
```

Comment:
Just flagging the use of `master` as the docker image tag... I suppose this is just following the git repo's master branch right now, and latest needs to be explicitly tagged (i.e. `latest` is semi-stable)?

Reply:
exactly. explained here https://github.com/pangeo-data/pangeo-docker-images#image-tagging-and-continuous-building